contributing/USER_PERSONAS.md

```diff
@@ -0,0 +1,207 @@
+# OpenAssetIO User Personas
+
+> Note: This page is a WIP, please add/suggest omissions through PR
+
+As a project, the majority of OpenAssetIO users are software developers.
+We count a "user" as someone who interacts with the code base and/or
+distribution, rather than with any higher-order functionality exposed by
+an application integrating the API.
+
+-   [Traditional DCC tool author](#traditional-commercial-dcc-tool-author)
```

Comment:
Maybe just "Commercial"? Easy to understand what's intended.

Reply:
Good call, thanks :)